streams/src/main/java/org/apache/kafka/streams/processor/DefaultPartitionGrouper.java

```diff
@@ -82,7 +82,10 @@ protected int maxNumPartitions(Cluster metadata, Set<String> topics) {
             List<PartitionInfo> partitions = metadata.partitionsForTopic(topic);
 
             if (partitions.isEmpty()) {
-                log.info("Skipping assigning topic {} to tasks since its metadata is not available yet", topic);
+
+                log.warn("Skipping assigning topic {} to tasks since its metadata is not available yet"
```

Comment:
The first line should be changed to `Skipping creating tasks for the topic group {} since topic {}'s metadata is not available yet; no tasks for this topic group will be assigned to any client.

Reply:
ack